proto/sysinfo.proto

```diff
@@ -479,3 +479,27 @@ message HardwareInfo {
     }];
 
 }
+
+// OSX Plist Start Calendar Interval
+
+message LaunchdStartCalendarInterval {
+  optional int32 minute = 1;
+  optional int32 hour = 2;
+  optional int32 day = 3;
+  optional int32 weekday = 4;
+  optional int32 month = 5;
+}
+
+// OSX Plist Files
+message LaunchdPlist {
+  optional string label = 1;
+  optional bool disabled = 2;
+  optional string username = 3;
```

Comment:
If there is no particular reason it might be better to call the fields in the proto the same as the plist fields so we dont need trivial backwards and forward conversions all the time.